gem/lib/pupperware/spec_helper.rb

```diff
@@ -236,6 +255,38 @@ def docker_volume_cp(src_path:, dest_volume:, dest_dir:, is_compose: true, uid:,
     end
   end
 
+  # takes a given certname, and generates certs to the dest_dir of dest_volume
+  # uses a transient pe-client-tools container with ssl.sh to generate certs
```

Comment:
This comment doesn't seem true.

Reply:
Ah thanks... it's not anymore. It started that way, but then I realized this is an OSS repo and we're not publishing that container publicly anywhere yet.

Reply:
Updated